pyquil/device.py

```diff
@@ -173,6 +176,137 @@ def gates_in_isa(isa):
     return gates
 
 
+class Specs(_Specs):
```

Comment:
Something to consider now or in the future:
Do we need a method that takes as input a qubit id and returns all the specs